### ISSUE TITLE
TypeError Cannot read property 'indexOf' of undefined, startsWith

### ISSUE THREAD

**Issue by charlag** [tutao/tutanota#851]
```
Client: Browser
Type: EXTERNAL
Tutanota version: 3.41.5
User agent: 
Mozilla/5.0 (Linux; Android 6.0.1; SM-G900F) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.80 Mobile Safari/537.36

Error message: 
Cannot read property 'indexOf' of undefined

Stacktrace: 
TypeError: Cannot read property 'indexOf' of undefined
at startsWith (https://mail.tutanota.com/common.js:1:3306)
at https://mail.tutanota.com/main.js:1:224365
at a (https://mail.tutanota.com/libs.js:26:73165)
at F._settlePromiseFromHandler (https://mail.tutanota.com/libs.js:26:46542)
at F._settlePromise (https://mail.tutanota.com/libs.js:26:47342)
at F._settlePromiseCtx (https://mail.tutanota.com/libs.js:26:47848)
at h (https://mail.tutanota.com/libs.js:26:1699)
at p (https://mail.tutanota.com/libs.js:26:1585)
at a._drainQueues (https://mail.tutanota.com/libs.js:26:3078)
at https://mail.tutanota.com/libs.js:26:1327
```

**Comment by charlag** [comment]
Seems to be exclusive to external users

**Comment by charlag** [comment]
Another stacktrace:
```

Error message: 
Unable to get property 'indexOf' of undefined or null reference

Stacktrace: 
TypeError: Unable to get property 'indexOf' of undefined or null reference
at startsWith (https://mail.tutanota.com/common.js:1:3288)
at Anonymous function (https://mail.tutanota.com/main.js:1:224357)
at a (https://mail.tutanota.com/libs.js:26:73149)
at F.prototype._settlePromiseFromHandler (https://mail.tutanota.com/libs.js:26:46383)
at F.prototype._settlePromise (https://mail.tutanota.com/libs.js:26:47337)
at F.prototype._settlePromiseCtx (https://mail.tutanota.com/libs.js:26:47843)
at h (https://mail.tutanota.com/libs.js:26:1697)
at p (https://mail.tutanota.com/libs.js:26:1585)
at a.prototype._drainQueues (https://mail.tutanota.com/libs.js:26:3078)
at drainQueues (https://mail.tutanota.com/libs.js:26:1325)
```

**Comment by armhub** [comment]
fixed with #885
